src/main/java/net/dries007/tfc/objects/items/metal/ItemProspectorPick.java

```diff
@@ -56,6 +59,13 @@ public EnumActionResult onItemUse(EntityPlayer player, World worldIn, BlockPos p
 
             if (!worldIn.isRemote)
             {
+                float falseNegativeChance = 0.4f; //Classic value was random(100) >= (60 + rank)
```

Comment:
I think we should keep the same calculation. It lowers the bar for "no false negatives", which is the only important milestone to the player (continuous incremental gains aren't rewarding, better stats on a new tier is). Thus false negative chance is `0.4 - skill.getTier().ordinal() * 0.1`

Reply:
Sorry, I made a mistake in the above calculation. The intent is for each tier to reduce false negatives by 10%, with all of master (not just 100% skill in master) giving no false negatives. You don't need any complicated logic here. The only change from above is `0.3 - skill.getTier().ordinal() * 0.1`